README.md

```diff
@@ -1632,6 +1632,9 @@ arXiv, 2020.
 - <a name="Haessig20fnins"></a>Haessig, G., Milde, M.B., Aceituno, P.V., Oubari, O., Knight, J.C., van Schaik, A., Benosman,  R. B., Indiveri, G.,  
 *[Event-Based Computation for Touch Localization Based on Precise Spike Timing](https://doi.org/10.3389/fnins.2020.00420)*,  
 Front. Neurosci. (2020), 14:420.
+<a name="Fariborz2020"></a> Naeini, F. B., Makris, D., Dongming, G., Zweiri, Y.,  
```

Comment:
Need to write a dash to start a new entry in the list, before the <a> tag.
Please see the formatting of previous entries and see the preview when writing the commit.
Thanks